modules/profile/manifests/st2server.pp

```diff
@@ -211,7 +211,7 @@
 
   class { '::nginx':
     package_name      => "${_nginx_package}",
-    service_restart   => '/etc/init.d/nginx configtest',
+    service_restart   => '/etc/init.d/nginx restart',
```

Comment:
I dunno if that is correct.

Maybe reload should be sufficient.

I also don't have enough background and context to know why this line the way it was.